quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/AbstractUpdatesCommand.java

```diff
@@ -94,11 +94,13 @@ void printError(String message) {
     }
 
     private void printPreviewWarning() {
-        printError("Warning! This command is preview and is not recommended for use in production. It may change or be removed at a future release.");
+        if (Profile.Feature.ROLLING_UPDATES_V1.getType() == Profile.Feature.Type.PREVIEW) {
```

Comment:
should we remove this warning?

Reply:
As `Profile.Feature.ROLLING_UPDATES_V1.getType() == Profile.Feature.Type.PREVIEW` evaluates to false (unless someone vetoes that one), it is not printed at the moment. 

And yes, the whole method can be removed until we re-add it once we introduce v2 :laughing: